### PR TITLE
Check path for maven capability

### PR DESCRIPTION
**Pull request by kelliejos**
Updated powershell script to look for maven in the path if maven home environment variable isn't set.

### CODE REVIEW

**Comment by TingluoHuang** [comment]
@madhurig

File: src/Misc/layoutbin/powershell/Add-MavenCapabilities.ps1 [hunk 1]
```diff
@@ -1,10 +1,40 @@
 [CmdletBinding()]
 param()
 
+function Get-MavenDirectoryFromPath {
```
Comment (madhurig) [inline src/Misc/layoutbin/powershell/Add-MavenCapabilities.ps1]: Get-Command mvn should get you the location without all this parsing. Please try it out.

Comment (madhurig) [reply src/Misc/layoutbin/powershell/Add-MavenCapabilities.ps1]: There is also a module you can use: https://github.com/Microsoft/vsts-agent/blob/f7e9a7ee42d0c30f0d31d449bb51d95bd6e9e0fb/src/Misc/layoutbin/powershell/CapabilityHelpers/CapabilityFunctions.ps1#L1

**Comment by kelliejos** [comment]
I updated the PR to use the helper capability function.

File: src/Misc/layoutbin/powershell/Add-MavenCapabilities.ps1 [hunk 2]
```diff
@@ -7,4 +7,9 @@ if (!$env:JAVA_HOME) {
     return
 }
 
-Add-CapabilityFromEnvironment -Name 'maven' -VariableName 'M2_HOME'
+if($env:M2_HOME) {
+    Add-CapabilityFromEnvironment -Name 'maven' -VariableName 'M2_HOME'
+} else {
+	Write-Host "M2_HOME not set. Checking in PATH"
```
Comment (TingluoHuang) [inline src/Misc/layoutbin/powershell/Add-MavenCapabilities.ps1]: extra tab. :)

**Pull request merged**